cmake/GitUtils.cmake

```diff
@@ -54,6 +54,9 @@ endif()
 #       <project name>_SOURCE_DIR
 #           top level source directory of the cloned project
 #
+#       <project name>_CLONE_RESULT
+#           Result of git_clone function. TRUE - success, FALSE - error
+#
```

Comment:
Do you think it would be better to let the user give the RESULT variable as an argument? I don't know how the conventions are in CMake.

Reply:
It will be more correct.
Maybe _SOURCE_DIR variable should be passed as an argument too?

Reply:
Yeah probably, but that would break backwards compatibility, so I can't change it now. I'll leave it to you to decide whether to make `_CLONE_RESULT` a parameter or not.